### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by kshitizsaini113**

### CODE REVIEW

File: _data/projects/Computer_Vision_with_Python [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+name: Computer Vision with Python
+desc: We aim to cver the following goals after completing the Computer Vision project : Understanding Computer Vision Applications, Understanding how to use OpenCV and how Python works with Iamages and Videos, Also, applying these skills in our projects.
```
Comment (shiftkey) [inline _data/projects/Computer_Vision_with_Python]: I believe the current problem with the build is the use of `:` here

>  `Computer Vision project : Understanding Computer Vision`

This is also very long - could it be simplified to a sentence or two?

File: _data/projects/Computer_Vision_with_Python [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+- python3
+- numpy
+- anaconda
+- opencv
+- contributions
+- image
+- imageprocessing
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/kshitizsaini113/Computer-Vision-with-Python/labels/up-for-grabs
```
Comment (shiftkey) [inline _data/projects/Computer_Vision_with_Python]: I see this label exists on the repository but there are no open issues - were you planning to create those and tag them with `up-for-grabs`?

**Review by shiftkey — COMMENTED** [review]
Thanks for submitting your project @kshitizsaini113!

File: _data/projects/Computer_Vision_with_Python (hunk 2 above)
Comment (kshitizsaini113) [reply _data/projects/Computer_Vision_with_Python]: Okay

**Review by kshitizsaini113 — COMMENTED** [review]
I have made the following changes

**Comment by shiftkey** [comment]
```
1 files found in projects directory which are not YAML files:
 - _data/projects/Computer_Vision_with_Python
```

Please ensure this project file has a `.yml` extension

**Pull request merged**